.evergreen/config/standard.yml.erb

```diff
@@ -184,7 +184,7 @@ buildvariants:
     matrix_spec:
       stress: on
       ruby: ["ruby-2.7"]
-      mongodb-version: ["4.4", '4.2', '4.0', '3.6']
+      mongodb-version: ['5.0', '4.4', '4.2', '4.0', '3.6']
```

Comment:
This is a good change to make, can you please put it into its own PR? You would also need to run `.evergreen/update-evergreen-configs` to update the yaml files that evergreen actually uses.

Reply:
Cool will do.